### DISCUSSION TITLE
Data Units

### DISCUSSION THREAD

**Discussion by GnomeTek** [theOehrly/Fast-F1#217] [Q&A] [answered]
Hello, 
I've been looking for units on the data and I cannot seem to find any. 
This was spurred by looking at the DRS graph and seeing multiple positions, which was unexpected. 

Is there a native units header in the data?

**Comment by theOehrly** [comment, marked as answer]
It's noted in the documentation of the api methods: 
https://theoehrly.github.io/Fast-F1/api.html

I agree that it's not straightforward to find though. I should really add this to the documentation of the Telemetry class or at least reference it. Or maybe make a shared table with that information.

**Reply by GnomeTek** [reply]
Ah there it is! 
Having it linked or brought up earlier would be a great addition to the documentation! 

Thanks for the quick reply, it's a neat system you've built!
